src/scores/probability/crps_impl.py

```diff
@@ -176,18 +176,22 @@ def crps_cdf(
 
     Given:
         - a predictive CDF `fcst` indexed at thresholds by variable x
-        - an observation in CDF form `obs_cdf` (i.e., :math:`\\text{obs_cdf}(x) = 0` if :math:`x < \\text{obs}` and 1 if :math:`x >= \\text{obs}`)
+        - an observation in CDF form `obs_cdf` (i.e., :math:`\\text{obs_cdf}(x) = 0` if \
+         :math:`x < \\text{obs}` and 1 if :math:`x >= \\text{obs}`)
         - a `threshold_weight` array indexed by variable x
 
     The threshold-weighted CRPS is given by:
-        - :math:`twCRPS = \\int_{-\\infty}^{\\infty}{[\\text{threshold_weight}(x) \\times (\\text{fcst}(x) - \\text{obs_cdf}(x))^2]\\text{d}x}`, over all thresholds x.
+        - :math:`twCRPS = \\int_{-\\infty}^{\\infty}{[\\text{threshold_weight}(x) \\times 
+        (\\text{fcst}(x) - \\text{obs_cdf}(x))^2]\\text{d}x}`, over all thresholds x.
         - The usual CRPS is the threshold-weighted CRPS with :math:`\\text{threshold_weight}(x) = 1` for all x.
```

Comment:
It could be that inline codeblocks cannot be split into multiple lines. Also I think that `_` i.e. as in `threshold_weight` gets parsed as subscript, and when I try to render it on online editors it spits out an error. I also feel like $\text{CRPS}_{tw}$ looks better than $twCRPS$. Here's a suggestion to try out:

```suggestion
    .. math::
        
        \\text{CRPS}_{tw} = \\int_{-\\infty}^{\\infty}{
            [\\text{threshold\\_weight}(x) \\cdot (\\text{fcst}(x) - \\text{obs\\_cdf}(x))^2]
            \\text{d}x}\\text{, over all thresholds $x$}.
        
    .. note::
        
        The usual CRPS is the threshold-weighted CRPS with :math:`\\text{threshold_weight}(x) = 1` for all x.
```

```math
        \text{CRPS}_{tw} = \int_{-\infty}^{\infty}{
            [\text{threshold\_weight}(x) \cdot (\text{fcst}(x) - \text{obs\_cdf}(x))^2]
            \text{d}x}\text{, over all thresholds $x$.}
```

Reply:
Generally in the literature, it is twCRPS, rather than CRPStw. E.g., see equation 3 in https://journals.ametsoc.org/view/journals/wefo/38/3/WAF-D-22-0161.1.xml

Reply:
This is merged now and rendering nicely on readthedocs. Multiline mathjax in a dot point just needs an extra backslash at the end of the split line, but it can be done.  I just re-run sphinx-build for my local testing, I haven't tried online editors but I find it reasonably efficient and close to what readthedocs will use (mostly just slightly different versions of things).